portfolio/src/main/java/com/google/sps/data/LongestTaskFirstScheduler.java

```diff
@@ -101,4 +113,69 @@ public SchedulingAlgorithmType getSchedulingAlgorithmType() {
 
     return scheduledTimeRangeOptional;
   }
+
+  /**
+   * Keeps splitting up the task to schedule the currently avaible free time ranges until all of the
```

Comment:
Consider updating the method comment to indicate that it updates scheduledTasks.

Reply:
Updated.